data/logs_model/logs_producer/splunk_logs_producer.py

```diff
@@ -32,23 +32,22 @@ def __init__(
             "scheme": url_scheme,
             "verify": verify_ssl,
         }
-        if ssl_ca_path:
-            # Create an SSLContext object for cert validation
-            context = ssl.create_default_context()
+        # Create an SSLContext object for cert validation
+        context = ssl.create_default_context()
+        if verify_ssl:
```

Comment:
nit: we could simplify conditional for easier readability
```suggestion
        context.check_hostname = True
        context.verify_mode = ssl.CERT_REQUIRED
        if not verify_ssl:
            context.check_hostname = False
            context.verify_mode = ssl.CERT_NONE
```

Reply:
I find it harder to read. But as that's the way it's written now, I don't mind keeping it as it is.